src/tests/simulation/src/cluster.rs

```diff
@@ -358,8 +363,10 @@ impl Cluster {
 
         // setup DNS and load balance
         let net = madsim::net::NetSim::current();
-        net.add_dns_record("etcd", "192.168.10.1".parse().unwrap());
```

Comment:
May check that `sqlite_data_dir` and `etcd_data_dir` are not set simutaneously.

Reply:
Fixed.